### ISSUE TITLE
[Bug] Autopilot does not activate the landing gear hotkey when landing

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#2358]
### Short Description of the issue:
When you land on a shipyard pad, the autopilot does not deploy landing gear if the hotkey brick is set to "Push" not "Switch"

### Steps to reproduce the issue:
Attempt to dock on a landing pad with a ship that has its landing gear set to Push (Pangolins Mistrel has this setup)

### Environment
- Starship EVO version: 20w23d
- Graphics Card: RX 580

**Comment by tsunamayo** [comment]
Yes there is nothing I can do for push mechanism, it needs to be set up to switch - I will make that more clear and also add a HUD icon. If at push then the AI has no way of understand when it is Up or Down.

**Comment by ProPeach** [comment]
Got it, yes a bit of extra help in the HUD would help a tonne
